src/vm/peimagelayout.cpp

```diff
@@ -591,7 +591,7 @@ PEImageLayout::EnumMemoryRegions(CLRDataEnumMemoryFlags flags)
 }
 #endif //DACCESS_COMPILE
 
-#if defined(_WIN64) && !defined(DACCESS_COMPILE)
+#if defined(_TARGET_64BIT_) && !defined(DACCESS_COMPILE)
 
```

Comment:
ConvertILOnlyPE32ToPE64 and ConvertILOnlyPE32ToPE64Worker is deadcode in .NET Core. It would be better to delete it